paasta_tools/utils.py

```diff
@@ -1590,6 +1591,13 @@ def get_cluster_autoscaling_draining_enabled(self) -> bool:
         :returns A bool"""
         return self.config_dict.get('cluster_autoscaling_draining_enabled', True)
 
+    def get_cluster_boost_enabled(self) -> bool:
+        """ Enable the cluster boost. Note boost that applies only to the resource cpus anyway.
```

Comment:
"Note that the boost only applies to the CPUs"

Reply:
done